src/libraries/System.Diagnostics.DiagnosticSource/tests/NativeAotTests/DiagnosticSourceEventSourceTests.cs

```diff
@@ -0,0 +1,109 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
+using System.Diagnostics.Tracing;
+
+/// <summary>
+/// Tests that using writing to a DiagnosticSource writes the correct payloads
+/// to the DiagnosticSourceEventSource.
+/// </summary>
+internal class Program
+{
+    private class TestEventListener : EventListener
```

Comment:
>TestEventListener [](http://example.com/codeflow?start=18&length=17)

Can we add a test cause ValueStringBuilder inside EnumeratePropertyFetch store more than 256 characters?

Reply:
ValueStringBuilder is used all over the codebase. I don't think this test would be valuable, as ValueStringBuilder is tested in many places.